core/src/model/ModelFactory.java

```diff
@@ -1,15 +1,26 @@
 package model;
 
-import model.enemies.*;
+import java.awt.*;
+import java.util.Arrays;
+import java.util.List;
 
 public class ModelFactory {
     public static Model makeModel(String mapName) {
-        Model model = new Model();
-        model.initialize(new TiledMapLoader(mapName));
-        model.addEnemy(new Cyclops(100, 100, 1, 1, 1, model.getWorld()));
-        model.addEnemy(new Cyclops(100, 200, 1, 1, 1, model.getWorld()));
-        model.addEnemy(new Cyclops(200, 100, 1, 1, 1, model.getWorld()));
-        model.addEnemy(new Mouse(200, 200,2, 1, 1, model.getWorld()));
+        TiledMapLoader mapLoader = new TiledMapLoader(mapName);
+        PlayerCharacter player = new PlayerCharacter(mapLoader.getMapUnitWidth() / 2, mapLoader.getMapUnitHeight() / 2, mapLoader.getWorld());
+
+        int xLeft = 100;
+        int xCenter = mapLoader.getMapUnitWidth() / 2;
+        int xRight = mapLoader.getMapUnitWidth() - 100;
+        int yBottom = 100;
+        int yCenter = mapLoader.getMapUnitHeight() / 2;
+        int yTop = mapLoader.getMapUnitHeight() - 100;
+        List<Point> spawnPoints = Arrays.asList(new Point(xLeft, yTop), new Point(xCenter, yTop), new Point(xRight, yTop),
+                                                new Point(xLeft, yCenter), new Point(xRight, yCenter),
+                                                new Point(xLeft, yBottom), new Point(xCenter, yBottom), new Point(xRight, yBottom));
```

Comment:
Won't the distance between spawnpoints be too great when we implement our larger map?

Reply:
This may have to change if a larger map is implemented.